projects/bp-gallery/src/components/provider/GrowthBookProvider.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  GrowthBook,
+  GrowthBookProvider as _GrowthBookProvider,
+} from '@growthbook/growthbook-react';
+import { PropsWithChildren, useEffect } from 'react';
+
+const growthbookApiHost = import.meta.env.VITE_REACT_APP_GROWTHBOOK_APIHOST;
+const growthbookClientKey = import.meta.env.VITE_REACT_APP_GROWTHBOOK_CLIENTKEY;
+
+const growthbook =
+  growthbookApiHost && growthbookClientKey
+    ? new GrowthBook({
+        apiHost: growthbookApiHost,
+        clientKey: growthbookClientKey,
+        enableDevMode: import.meta.env.MODE === 'development',
+        trackingCallback: (experiment, result) => {
+          const w: any = window;
+          const _paq: Array<any> = (w._paq = w._paq || []);
+          _paq.push([
+            'trackEvent',
+            'ExperimentViewed',
+            experiment.key,
+            'v' + String(result.variationId),
+          ]);
+        },
+        onFeatureUsage: (featureKey, result) => {
+          const w: any = window;
+          const _paq: Array<any> = (w._paq = w._paq || []);
+          _paq.push(['trackEvent', 'ExperimentViewed', featureKey, 'v' + String(result)]);
```

Comment:
Can you distinguish features from "experiments" above when they have the same name? What are experiments?